### PR TITLE
feat(demo): Optionally load LittleFS image file to simulated flash

### PR DESCRIPTION
**Pull request by tomods**
This adds the capability to load a LittleFS image that was externally created. This is needed for MicroPython to automatically execute scripts, and massively facilitates MicroPython development on rp2040js.

Currently, this is put directly into the existing demo `micropython-run.ts` – I don't think it is too intrusive for the simple example. I could also move it into a separate file.

### CODE REVIEW

**Comment by tomods** [comment]
For reference, I am currently using this https://github.com/tomods/GrinderController/blob/358ad3e0f795d8cc0bdf4f21bb35f806871d433f/tools/create_littlefs_image.py to generate my LittleFS images.

**Comment by urish** [comment]
> For reference, I am currently using this https://github.com/tomods/GrinderController/blob/358ad3e0f795d8cc0bdf4f21bb35f806871d433f/tools/create_littlefs_image.py to generate my LittleFS images.

Worth adding this to the README (under "Run the demo project"). While at it, I'd also mention Ctrl+X to quit.

Mind adding those into the PR

File: README.md [hunk 1]
```diff
@@ -19,23 +19,45 @@ If you want to develop your own application using the Raspberry Pi Pico simulato
...
+and enjoy the MicroPython REPL! Quit the REPL with Ctrl+X.
 
-You can replace rp2-pico-20210902-v1.17.uf2 any recent MicroPython or CircuitPython release built for the RP2040.
+You can replace rp2-pico-20210902-v1.17.uf2 with any recent MicroPython or CircuitPython release built for the RP2040.
+
+With MicroPython – and probably also CircuitPython – you can use the filesystem on the Pico. This becomes useful as more than one script file is used in your code. Just put a [LittleFS](https://github.com/littlefs-project/littlefs) formatted filesystem image called `littlefs.img` into the rp2040js root directory, and your `main.py` will be automatically started from there.
+
+A simple way to create a suitable LittleFS image containing your script files is outlined in [create_littlefs_image.py](https://github.com/tomods/GrinderController/blob/358ad3e0f795d8cc0bdf4f21bb35f806871d433f/tools/create_littlefs_image.py).
+So, using [littlefs-python](https://pypi.org/project/littlefs-python/), you can do the following:
+```
```
Comment (urish) [inline README.md:46]: Please change to ```python

so we get syntax highlighting

**Comment by urish** [comment]
Excellent, thanks!

**Pull request merged**